src/core/content-snapshot.js

```diff
@@ -18,7 +18,7 @@ export class ContentSnapshot extends Phaser.Image {
      * @extend Phaser.Image
      */
     constructor(game, x, y, noStage) {
-        const texture = new Phaser.RenderTexture(game, game.width, game.height);
+        const texture = new Phaser.RenderTexture(game, game.width, game.height, "", game.renderer.resolution);
```

Comment:
Hey @hardylr, thanks for submitting this PR!

I took a quick look, over the docs, to have a reference for your issue, and it looks like the fourth argument is the `key`, the fifth is the `scaleMode` and the sixth the `resolution`. I think you need to declare the `game.renderer.resolution` to be the sixth argument.

```js
const texture = new Phaser.RenderTexture(game, game.width, game.height, "", Phaser.scaleModes.DEFAULT, game.renderer.resolution);
```

Reply:
Hey @cristianbote , you are correct. I messed up the argument order. Will fix it.